compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -403,6 +411,31 @@ fn link_rlib<'a>(
         ab.add_file(&trailing_metadata);
     }
 
+    // add all native rlib dependencies
```

Comment:
```suggestion
    // Add all native rlib dependencies.
```
Style nit: comments like this use full sentences with a capital letters and dot.